src/EnergyPlus/HVACFan.cc

```diff
@@ -324,6 +333,7 @@ namespace HVACFan {
         // ANSI/AMCA Standard 207-17: Fan System Efficiency and Fan System Input Power Calculation, 2017.
         // AANSI / AMCA Standard 208 - 18: Calculation of the Fan Energy Index, 2018.
 
+        assert(DataEnvironment::StdRhoAir > 0.0);
```

Comment:
Several unit tests were causing the fan inits to fire and call the FEI calc without setting `StdRhoAir` which is initialized to zero. So the FEI was returning `nan`. Add this assert to catch that.